configs/sites/aws-pcluster/packages.yaml

```diff
@@ -0,0 +1,137 @@
+packages:
+  all:
+    compiler: [intel@2021.4.0]
+    providers:
+      mpi: [intel-oneapi-mpi@2021.4.0]
+
+### MPI, Python, MKL
+  intel-oneapi-mpi:
+    buildable: false
+    externals:
+    - spec: intel-oneapi-mpi@2021.4.0%intel@2021.4.0
+      prefix: /opt/intel
+      modules:
+      - intelmpi
+  python:
+    buildable: false
+    externals:
+    - spec: python@3.7.10+bz2+ctypes+dbm+ensurepip+lzma+nis+pyexpat+pythoncmd+readline+sqlite3+ssl+tix+tkinter+uuid+zlib
+      prefix: /opt/parallelcluster/pyenv/versions/3.7.10/envs/awsbatch_virtualenv
+    - spec: python@2.7.18+bz2+ctypes+dbm+ensurepip+nis+pyexpat+pythoncmd+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
+      prefix: /usr
+    - spec: python@3.8.10+bz2+ctypes+dbm+ensurepip+lzma+nis+pyexpat~pythoncmd+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
+      prefix: /usr
```

Comment:
Why are there 3 python versions, ie won't this potentially confuse the build process and keeping the python versions matching between building and running applications?

Allowing for a python version 2 seems dangerous since we moved JEDI to python3 a ways back.

Reply:
I can remove the two older ones, spack-stack won't use them because the latest takes precedence. But it' cleaner, so yes!